.github/workflows/pnp-build-autopilot-sink.yml

```diff
@@ -0,0 +1,226 @@
+name: Build and push sink to staging
```

Comment:
This workflow can be used to deploy sink not only yo autopilot, but also to cloud run. I would remove `autopilot` from the name and give it more generic name like `pnp-pubsub-sink-build`